js/index.js

```diff
@@ -1 +1,110 @@
 // Your code goes here
+//all selctors
+let mainNav = document.querySelector('.main-navigation');
+let logoHead = document.querySelector('.logo-heading');
+let navAnchors = document.querySelectorAll('.nav-link');
+let header = document.querySelector('.main-navigation');
+let allImgs = document.querySelectorAll('img');
+let footer = document.querySelector('.footer');
+let button = document.querySelectorAll('.btn');
+
+
+//header change background, alert, and transform logo
+
+mainNav.addEventListener('mouseenter', (event) => {
+    mainNav.style.backgroundColor = 'teal';
+})
+
+logoHead.addEventListener('click', (event) => {
+    alert('Are you ready for FUN?!')
+})
+document.addEventListener("scroll", function (e) {
+    logoHead.style.transform = "rotate(180deg)";
+})
+
+
+//nava's create hover
+navAnchors.forEach(function (index, i) {
+    navAnchors[i].addEventListener('mouseover', function (e) {
+        e.target.style.color = '#fcc100';
+        e.target.style.textDecoration = 'underline';
+    });
+});
+
+navAnchors.forEach(function (index, i) {
+    navAnchors[i].addEventListener('mouseout', function (e) {
+        e.target.style.color = '';
+        e.target.style.textDecoration = 'none';
+    });
+});
+
+
+//Prevent anchor tags in nav from refreshing page
+
+document.querySelectorAll('nav a').forEach(function (link) {
+    link.addEventListener("click",
+        function (e) {
+            e.preventDefault();
+        }
+    )
+});
+
+//right click on header
+header.addEventListener('contextmenu', function (e) {
+    alert("What are you doing?");
+});
+
+//for purposes of using stopPropagation when copy ps vs hs
+document.querySelectorAll('.container ').forEach(function (heading) {
+    heading.addEventListener("copy",
+        function (e) {
+            alert("I know my headings are FUN!");
+        }
+    )
+});
+document.querySelectorAll('p').forEach(function (text) {
+    text.addEventListener("copy",
+        function (e) {
+            alert("Give me props!");
+            e.stopPropagation();
+
+        }
+    )
+});
+//imgs event listener to skew based on single or double click
+
+allImgs.forEach((img) => {
+    img.addEventListener('click', function (e) {
+        e.target.style.transform = 'skewX(-15deg)';
```

Comment:
This is neat, but one thing to look out for is messing with images can often result in those images get their z index bumped up. As a result your images appear to be in front of your header background. You can fix this by giving your header background a higher z-index